### DISCUSSION TITLE
How to use this for odd numbers?

### DISCUSSION THREAD

**Discussion by FurkanToprak** [samuelmarina/is-even#129] [Q&A]
Help me

**Comment by toydotgame** [comment]
If it returns `true` for an even number, it'll return `false` for an odd one. I don't know JavaScript, but take this pseudocode for the concept:
```
if(is_even(17) == true) {
    // This code will run if the number is even.
}

if(is_even(17) == false) {
    // This code will run if the given number is odd.
}
```

**Reply by FurkanToprak** [reply]
Thanks, this is perfect javascript!

**Comment by samuelmarina** [comment]
Check [@samuelmarina/is-odd](https://github.com/samuelmarina/is-odd), we're all about the community here
